editor/src/clj/editor/gui.clj

```diff
@@ -2419,6 +2451,11 @@
   (property id g/Str (default "") ; No protobuf counterpart.
             (dynamic visible (g/constantly false)))
 
+  (input basic-gui-scene-info BasicGuiSceneInfo)
+  (output basic-gui-scene-info BasicGuiSceneInfo (gu/passthrough basic-gui-scene-info))
+  (input costly-gui-scene-info CostlyGuiSceneInfo)
+  (output costly-gui-scene-info CostlyGuiSceneInfo (gu/passthrough costly-gui-scene-info))
+
```

Comment:
This is on the `NodeTree`, which would pass all this stuff down to the hosted `GuiNodes`.